lib/ft817/ft817.h

```diff
@@ -409,7 +409,7 @@ class FT817
 		// vars
 		unsigned long freq;		// frequency data as a long
 		byte mode;					// last mode read
-		unsigned char buffer[5];	// buffer used to TX and RX data to the radio
+		byte buffer[5];	// buffer used to TX and RX data to the radio

```

Comment:
bytes were all being stored as 255 or 0 when this was unsigned char... changed buffer's type to byte and the bytes received from the radio are stored correctly